packages/react-components/react-menu/library/src/contexts/menuContext.ts

```diff
@@ -46,6 +46,7 @@ export type MenuContextValue = Pick<
   | 'inline'
   | 'checkedValues'
   | 'onCheckedValueChange'
+  | 'safeZone'
```

Comment:
`safeZone` is a React element but it's memoized, so it won't cause context updates